Yona/Yona/Others/SMSValidPasscodeLogin/Login/LoginViewController.swift

```diff
@@ -206,15 +204,17 @@ extension LoginViewController: CodeInputViewDelegate {
         let passcode = KeychainManager.sharedInstance.getPINCode()
         if code ==  passcode {
             authenticatedSuccessFully()
+            UserDefaults.standard.set(0, forKey: YonaConstants.nsUserDefaultsKeys.numberOfLoginAttempts) // resetting number of login attempts to 0
```

Comment:
If you move the call into a method, you don't need to add a comment:
```suggestion
            setFailedLoginAttempts(0)
```

Reply:
Updated code accordingly.